overlay/usr/local/bin/gameserver-init

```diff
@@ -4,56 +4,67 @@ GAMEUSER="gameuser"
 LOGFILE="/var/log/gameserver/install.log"
 GAME_REPO_DIR="/root/gameservers"
 
-function setup_game_user {
-    . /etc/default/inithooks
-    $INITHOOKS_PATH/bin/setpass.py $GAMEUSER --pass="$APP_PASS"
+fatal() { echo "FATAL: $@"; exit 1; }
+
+clean_finish() {
+    sed -i "\|^$GAMEUSER| s|^|#|" /etc/sudoers
+    if [[ $1 -ne 0 ]]; then
+        echo
+        fatal "Install was interrupted and/or failed."
+    else
+        echo "Installation complete."
+    fi
 }
 
-function select_game_interactive {
+select_game_interactive() {
     python3 "/usr/local/bin/gameserver-menu.py" "$GAME_REPO_DIR"
 }
 
-function select_game_auto {
+select_game_auto() {
     cp "$GAME_REPO_DIR/games/$GAME/game_properties.sh" "/etc/gameserver/gameserver"
 }
 
-function select_game {
+select_game() {
     if [ ! -d "/etc/gameserver" ] || [ ! -f "/etc/gameserver/gameserver" ]; then
         mkdir -p "/etc/gameserver"
 
         if [ -n "$GAME" ]; then
             select_game_auto
         else
             # Exit if not running interactively, game will be selected on first login
-            [ -z "`stty size`" ] && exit 0
+            echo $(stty size) >> /root/tty.size
```

Comment:
Why is this value being written to `/root/tty.size` ?

Reply:
Oops. I added that for testing and forgot to remove it. I've just removed it and repushed.